packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -40,7 +40,7 @@ class RenderProxyBox extends RenderBox with RenderObjectWithChildMixin<RenderBox
   /// Proxy render boxes are rarely created directly because they simply proxy
   /// the render box protocol to [child]. Instead, consider using one of the
   /// subclasses.
-  RenderProxyBox([RenderBox child = null]) {
+  RenderProxyBox([RenderBox child = null]) { //ignore: avoid_init_to_null
```

Comment:
If I remove `= null` I get the following issues from Analysis Server:

```
$ flutter analyze --flutter-repo --watch
  error • Invalid override. The type of 'RenderTransform.applyPaintTransform' ('(RenderBox, Matrix4) → void') isn't a subtype of 'RenderProxyBoxMixin.applyPaintTransform' ('(RenderObject, Matrix4) → void'). • packages/flutter/lib/src/rendering/proxy_box.dart:1665:3
  error • Invalid override. The type of 'RenderFittedBox.applyPaintTransform' ('(RenderBox, Matrix4) → void') isn't a subtype of 'RenderProxyBoxMixin.applyPaintTransform' ('(RenderObject, Matrix4) → void'). • packages/flutter/lib/src/rendering/proxy_box.dart:1799:3
  error • Invalid override. The type of 'RenderFractionalTranslation.applyPaintTransform' ('(RenderBox, Matrix4) → void') isn't a subtype of 'RenderProxyBoxMixin.applyPaintTransform' ('(RenderObject, Matrix4) → void'). • packages/flutter/lib/src/rendering/proxy_box.dart:1865:3
  error • Invalid override. The type of 'RenderFollowerLayer.applyPaintTransform' ('(RenderBox, Matrix4) → void') isn't a subtype of 'RenderProxyBoxMixin.applyPaintTransform' ('(RenderObject, Matrix4) → void'). • packages/flutter/lib/src/rendering/proxy_box.dart:3309:3
  error • Invalid override. The type of '_RenderSingleChildViewport.attach' ('(PipelineOwner) → void') isn't a subtype of 'AbstractNode.attach' ('(Object) → void'). • packages/flutter/lib/src/widgets/single_child_scroll_view.dart:231:3
  error • Invalid override. The type of '_RenderSliverPrototypeExtentList.attach' ('(PipelineOwner) → void') isn't a subtype of 'AbstractNode.attach' ('(Object) → void'). • packages/flutter/lib/src/widgets/sliver_prototype_extent_list.dart:151:3
6 issues found • analyzed 1145 files, 5.17 seconds
```

But directly with analyzer package:

```
$ flutter analyze --flutter-repo
Building flutter tool...
Analyzing 1144 files...
No analyzer warnings! (ran in 22.9s; 1 public member lacks documentation)
```

This looks like an issue in analysis server. /cc @bwilkerson @pq @danrubel